### ISSUE TITLE
Contrast too low for WCAG standards

### ISSUE THREAD

**Issue by itzsaga** [undergroundjs/website#1]
I randomly ran a contrast check on the website as I was curious. Turns out that the orange background #f48705 with white text of "Buy Tickets Now" has a contrast of 2.51:1 which is below the recommended 3:1 for large text. I tested with https://color.a11y.com/Contrast/ Consider changing that color to meet the requirement.

**Comment by kwelch** [comment]
Thanks! Fixed this up by switching to black. Going to look into adding a hook to verify these on commits.
